### PR TITLE
Update 22.vaccines.md

### PR DESCRIPTION
**Pull request by nilswellhausen**
### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

**Comment by nilswellhausen** [comment]
Forgot to add this ref after the information on viral vector vaccines approved/in trial: [@doi:10.1128/CVI.00298-16]

File: content/22.vaccines.md [hunk 1]
```diff
@@ -168,7 +168,11 @@ Based on the efficacy and safety information released, the vaccine was approved
 As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19].
 
 ### Viral Vector Vaccines
-
+As discussed earlier, delivery and presentation of antigens are key to induce immunity against SARS-CoV2.
+Viral vectors have emerged as a safe and efficient method to deliver foreign substances to induce an efficient immune response [@doi:10.1128/CVI.00298-16].
+There are several viral vector vaccines that are available for veterinary use [@doi:10.3390/vaccines8040680].
+While there is currently only one FDA-approved viral vector vaccine for the use in humans against Ebola [doi:10.1016/j.cell.2020.03.011], several Phase I and Phase II clinical trial are ongoing.
+Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV2. Here we review the current state of the art for developing viral vector vaccines against SARS-CoV2.
```
Comment (rando2) [inline content/22.vaccines.md:175]:
```suggestion
Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV-2.
Here we review the current state of the art for developing viral vector vaccines against SARS-CoV-2.
```

File: content/22.vaccines.md [hunk 2]
```diff
@@ -168,7 +168,11 @@ Based on the efficacy and safety information released, the vaccine was approved
 As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19].
 
 ### Viral Vector Vaccines
-
+As discussed earlier, delivery and presentation of antigens are key to induce immunity against SARS-CoV2.
+Viral vectors have emerged as a safe and efficient method to deliver foreign substances to induce an efficient immune response [@doi:10.1128/CVI.00298-16].
+There are several viral vector vaccines that are available for veterinary use [@doi:10.3390/vaccines8040680].
+While there is currently only one FDA-approved viral vector vaccine for the use in humans against Ebola [doi:10.1016/j.cell.2020.03.011], several Phase I and Phase II clinical trial are ongoing.
```
Comment (rando2) [inline content/22.vaccines.md:174]:
```suggestion
While there is currently only one FDA-approved viral vector vaccine for the use in humans against Ebola [doi:10.1016/j.cell.2020.03.011], several Phase I and Phase II clinical trial are ongoing [@doi:10.1128/CVI.00298-16].
```
@nilswellhausen is this where you wanted it?

**Comment by rando2** [comment]
@ajlee21 would you be able to review this PR?

File: content/22.vaccines.md [hunk 3]
```diff
@@ -168,7 +168,11 @@ Based on the efficacy and safety information released, the vaccine was approved
 As of December 11, 2020, the United States FDA approved this vaccine under an emergency use authorization [@url:https://www.fda.gov/news-events/press-announcements/fda-takes-key-action-fight-against-covid-19-issuing-emergency-use-authorization-first-covid-19].
 
 ### Viral Vector Vaccines
-
+As discussed earlier, delivery and presentation of antigens are key to induce immunity against SARS-CoV2.
+Viral vectors have emerged as a safe and efficient method to deliver foreign substances to induce an efficient immune response [@doi:10.1128/CVI.00298-16].
```
Comment (ajlee21) [inline content/22.vaccines.md:172]:
```suggestion
Viral vectors have emerged as a safe and efficient method to deliver foreign substances to induce an efficient immune response [@doi:10.1128/CVI.00298-16].
These vaccines use a different virus to deliver genetic material from the target virus of interest.
The vaccine uses the host machinery to construct antigen(s) from the transported genetic material.
```

File: content/22.vaccines.md (hunk 1 above)
Comment (ajlee21) [inline content/22.vaccines.md:175]:
```suggestion
These vaccines are able to induce both an antibody and cellular response, however the response is limited due to the immunogenicity of the viral vector used [@doi:10.1016/j.coi.2011.03.006; @doi:10.1038/mt.2009.130].
Various viral vector platforms including Poxviruses, Adenoviruses, and Vesicular Stomatitis Viruses (VSV) are being developed for treating numerous infectious diseases such as Malaria, Ebola, HIV, and more recently, SARS-CoV2. Here we review the current state of the art for developing viral vector vaccines against SARS-CoV2.
```

**Review by ajlee21 — APPROVED** [review]
This addition looks good to me. 

I tried to add a bit more on how these viral vectors work

**Comment by nilswellhausen** [comment]
Looking good! ready to merch.

**Pull request merged**